src/foam/lib/csv/CSVEscapeStringParser.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.lib.csv;
+
+import foam.lib.parse.*;
+
+
+public class CSVEscapeStringParser implements Parser {
```

Comment:
Can we document what makes this different from the JSON string parser?

Reply:
Yes, I will do that

Reply:
Should \r \n and others be interpreted to real carriage returns and line feeds?

Reply:
No, \r\n will be parsed inside the CSVEscapeStringParser. I have modified the CSVNormalStringParser to avoid \r\n.